Updated with product demo videos requested by Sri (#1034)
harness/developer-hub: docs/continuous-delivery/onboard-cd/cd-concepts/services-and-environments-overview.md

If you are new to Harness, please review [Harness key concepts](../../../getting-started/learn-harness-key-concepts.md) and [CD pipeline modeling overview](cd-pipeline-modeling-overview.md).

> ## Product Demo - Services

Reviewer: ```suggestion
## Visual summary - Services
```

Reviewer: Video clarity is not great

<!-- Video:
https://harness-1.wistia.com/medias/xtmi0c0z95-->
<docvideo src="https://harness-1.wistia.com/medias/xtmi0c0z95" />

> ### Product Demo - Service and Environments at the Account and Org Level

Reviewer: ```suggestion
### Visual summary of creating service and environments at an account and org level
```
